scripts/needed-auto-tfvars-files/aks/misc.auto.tfvars.example

```diff
@@ -0,0 +1,22 @@
+resource_groups = { # rg
+  azure_kubernetes_service = {
+    tags = { "enclosed resource" = "open source aks" }
+  }
+  # azure_log_analytics_workspace = {
```

Comment:
Please delete commented-out code here and elsewhere.

Reply:
Fixed